Assurance Cases.md

```diff
@@ -75,14 +75,18 @@ As for Home Assitant official add-ons, we presume that they are secure and if a
 #### [Return to top](#assurance-case-list)
 
 ### Assurance Case 5
-#### .red[TODO] Insert Assurance Case 5 Name
+#### Home Assistant minimizes information disclosure during communication between remote networks and local instances.
 
-![Assurance Case 5 Diagram:](/images/)
-.red[TODO] Insert Assuracne Case 5 Image
+![Assurance Case 1 Diagram:](/images/Assurance_Case_5.PNG)
 
 #### Evidence for Assurance Case 5
+Home Assistant offers users to have access to their Home Assistant integrations outside of the user's local network. This feature is developed by the founders of Home Assistant and serves Home Assistant Cloud (Otherwise known as [Nabu Casa](https://www.nabucasa.com/)) as an optional add-on on top of the traditional Home Assistant integration. The setup process for expanding your integration to allow remote access is rather seamless, with various amounts of [documentation supporting it](https://www.nabucasa.com/config/). To enable the remote access feature, while connected to your Home Assistant locally on a user's network, navigate to the cloud options and remote control is a simple toggle switch, from their Home Assistant takes care of the rest. This feature allows users to enhance their Home Assistant integration with the ability to manage their home outside of their network. However, with the addition of this feature the risk of user's information being hijacked in transit from remote devices to an at home network increases.  
```

Comment:
The second to last sentence ... navigate to the cloud options and remote control is a simple toggle switch ... reads a little weird